test/lib/npm.js

```diff
@@ -0,0 +1,143 @@
+import * as npm from "../../lib/npm"
+import expect from "expect"
+import mappings from "../../config/mappings"
+import snap from "snap-shot-it"
+let specCompliantBowerConfig = require("o-spec-compliant-bower-config/bower.json")
+
+let props = {
+	name: "yeet",
+	version: "0.0.0-yeet",
+}
+
+describe("merge manifests", () => {
+	it("overwrites deps from existing with generated", () => {
```

Comment:
From reading the expected result, it looks like mergeMenifests will merge the dependencies from generated into existing, not overwrite the dependencies from existing with generated.
```suggestion
	it("merges deps from existing with generated", () => {
```

Reply:
oh, this is badly worded. I mean that it will _prefer_ deps from generated over existing

Reply:
```suggestion
	it("prefers deps from the generated over the existing", () => {
```